addons/account/static/src/interactions/account_portal_sidebar.js

```diff
@@ -0,0 +1,58 @@
+import { PortalSidebar } from "@portal/src/interaction";
+import { registry } from "@web/core/registry";
+
+import { scrollTo } from "@web/core/utils/scrolling";
+
+export class AccountPortalSidebar extends PortalSidebar {
+    static selector = ".o_portal_invoice_sidebar";
+    dynamicContent = {
+        _window: { "t-on-resize": this.updateIframeSize },
+        ".o_portal_invoice_print": { "t-on-click.prevent.withTarget": this.onClickPrint },
+    };
+
+    setup() {
+        super.setup();
+        this.invoiceHTMLEl = this.el.querySelector("iframe#invoice_html");
+    }
+
+    start() {
+        super.start();
+        const iframeDoc = this.invoiceHTMLEl.contentDocument || this.invoiceHTMLEl.contentWindow.document;
+        if (iframeDoc.readyState === 'complete') {
+            this.updateIframeSize();
+        } else {
+            this.addListener(this.invoiceHTMLEl, "load", this.updateIframeSize);
+        }
+    }
+
+    /**
+     * Called when the iframe is loaded or the window is resized on customer portal.
+     * The goal is to expand the iframe height to display the full report without scrollbar.
+     */
+    updateIframeSize() {
+        const wrapwrapEl = this.invoiceHTMLEl.contentDocument.querySelector("div#wrapwrap");
+        this.invoiceHTMLEl.style.height = 0;
+        this.invoiceHTMLEl.style.height = wrapwrapEl.scrollHeight;
+        // scroll to the right place after iframe resize
+        const isAnchor = /^#[\w-]+$/.test(window.location.hash)
+        if (!isAnchor) {
+            return;
+        }
+        const targetEl = window.location.hash
```

Comment:
`location.hash` is not a DOM element, I think you need to query it by ID. (and missing `;`)